### DISCUSSION TITLE
data 콤마관련

### DISCUSSION THREAD

**Discussion by jewon7377** [nhn/tui.chart#715] [Q&A] [answered]
안녕하세요.
데이터 처리를 할때 예를들어 가격표시부분에서 콤마처리는 어떻게 하는지 문의드립니다.

**Comment by jwlee1108** [comment, marked as answer]
@jewon7377 

Hello, I missed your question. I'm sorry for the late replying.
I think if you are using a typical chart, please refer to the links below and try the `formatter` option.

https://github.com/nhn/tui.chart/blob/main/docs/en/common-axes.md#formatter
https://github.com/nhn/tui.chart/blob/main/docs/en/common-tooltip.md#formatter

FYI, here is the sample code with regexp. I used this regexp in this [link](https://hianna.tistory.com/441) to make the sample code.

```js
yAxis: {
  title: 'Temperature (Celsius)',
    label: {
      formatter: (value) => value.toString().replace(/\B(?<!\.\d*)(?=(\d{3})+(?!\d))/g, ','),
    },
  },
tooltip: {
  formatter: (value) => value.toString().replace(/\B(?<!\.\d*)(?=(\d{3})+(?!\d))/g, ','),
},
```
